src/java.management/share/classes/javax/management/modelmbean/DescriptorSupport.java

```diff
@@ -1057,6 +1064,7 @@ private static String quote(String s) {
         return buf.toString();
     }
 
+    @SuppressWarnings("removal")
```

Comment:
Can we add a comment to indicate that XMLParseException is deprecated for removal?